julia-mode.el

```diff
@@ -48,19 +48,15 @@
 
 (defcustom julia-indent-offset 4
   "Number of spaces per indentation level."
-  :type 'integer
-  :group 'julia)
+  :type 'integer)
 
 (defface julia-macro-face
   '((t :inherit font-lock-preprocessor-face))
-  "Face for Julia macro invocations."
-  :group 'julia-mode)
```

Comment:
also here, and the one below

Reply:
This one was actually deleted because `julia-mode` was a group only created anonymously by these defgroup invocations. I'm assuming it was a mistake and they were meant to be members of `julia` group instead.